jetty-websocket/websocket-javax-client/src/main/java/module-info.java

```diff
@@ -23,11 +23,11 @@
 module org.eclipse.jetty.websocket.javax.client
 {
     exports org.eclipse.jetty.websocket.javax.client;
+    exports org.eclipse.jetty.websocket.javax.client.internal to org.eclipse.jetty.websocket.javax.server;
```

Comment:
@sbordet This is a weird case, I needed this because `websocket-javax-server` needs to extend on `websocket-javax-client` but we still don't want to transitively export our classes which have `websocket-core` in the signature.

This gives an error 
```
Warning:(26, 99) java: module not found: org.eclipse.jetty.websocket.javax.server
```
I'm guessing this is because this module is complied before the `websocket-javax-server` module or something like that.